test-addon/src/background.js

```diff
@@ -81,9 +82,9 @@ class StudyLifeCycleHandler {
  * @returns {Promise<void>} Nothing
  */
 async function onEveryExtensionLoad() {
-  new StudyLifeCycleHandler();
-  const studySetup = await getStudySetup();
-  await browser.study.setup(studySetup);
+  // new StudyLifeCycleHandler();
+  // const studySetup = await getStudySetup();
+  // await browser.study.setup(studySetup);
```

Comment:
Line 37 we can comment/remove all console.* as long as we have sufficient logging of add-on install and uninstall in api.js

Reply:
@gregglind  I assume studyInfo.timeUntilExpire is calculated and different from studyInfo.expire defined inside studySetup.js?